### PR TITLE
Feature/get primary sensor data

### PR DESCRIPTION
**Pull request by shanlyMIDE**
I need help being able to reference `get_doc` (I thought I did it correctly with `from .files import get_doc` but it doesn't seem to work. In any case this should satisfy https://github.com/MideTechnology/endaq-python/issues/32

Here is a notebook with some test examples (once I can fix the above): https://colab.research.google.com/drive/1NCte1dvwSaGxKZYN9JtNouA3mitwR61k#scrollTo=VDH-aRtaXuVd
image

### CODE REVIEW

**Comment by CFlaniganMide** [comment]
The issue is that `.info` is importing `.files`, and `.files` is importing `.info`.  The best way to fix this kind of issue really depends on the situation, but in this case the best course might be to move the `parse_time` function out of `.info` and into `.files`.

**Comment by StokesMIDE** [comment]
> The issue is that `.info` is importing `.files`, and `.files` is importing `.info`. The best way to fix this kind of issue really depends on the situation, but in this case the best course might be to move the `parse_time` function out of `.info` and into `.files`.

IMHO, the function `parse_time()` should probably be moved to `endaq.ide.util`.

**Comment by CFlaniganMide** [comment]
> > The issue is that `.info` is importing `.files`, and `.files` is importing `.info`. The best way to fix this kind of issue really depends on the situation, but in this case the best course might be to move the `parse_time` function out of `.info` and into `.files`.
> 
> IMHO, the function `parse_time()` should probably be moved to `endaq.ide.util`.

Yeah, that makes more sense.

File: endaq/ide/info.py [hunk 1]
```diff
@@ -328,3 +262,83 @@ def to_pandas(
...
+        doc = get_doc(name)
+        
+    #Get Channels of the Measurement Type
+    channels = get_channel_table(doc, measurement_type).data
+    
+    #Raise error if measurement type isn't in the file
+    if len(channels)==0:
+        error_str = f'measurement type "{measurement_type}" is not included in this file'
+        if force_data_return:
+            print(error_str)
```
Comment (StokesMIDE) [inline endaq/ide/info.py:326]: This should be removed, or changed to a warning or logger message

**Review by StokesMIDE — CHANGES_REQUESTED** [review]

**Comment by StokesMIDE** [comment]
This should probably get tests before fully approving/merging. Thoughts?

**Comment by shanlyMIDE** [comment]
I'm going to merge now that @StokesMIDE added the tests and approved it. I'm motivated to get it in because we had some engineers here at Mide looking to see the docs for it because they liked it when I showed it to them in an example script!

**Comment by shanlyMIDE** [comment]
@StokesMIDE  and @CrepeGoat  do you know when the docs will update? I had been motivated to merge it into dev to show Stan here but I don't see it yet in the docs: https://docs.endaq.com/en/development/endaq/ide.html

I'm not sure if we need to change something in the code to tell the docs to display that function?

**Comment by CrepeGoat** [comment]
you're looking for this, right? https://docs.endaq.com/en/development/endaq/ide.html#endaq.ide.get_primary_sensor_data

> I had been motivated to merge it into dev to show Stan here
Generally I recommend adding the branch you're working on as a "build source" (not a real term, don't quote me :P) for readthedocs and inspecting the docs changes *before* merging branches. I'll put up a Confluence page on how to do that.

**Comment by shanlyMIDE** [comment]
well that's the right link URL but the function isn't shown?
image

**Comment by CrepeGoat** [comment]
ohhhh that's a weird one. I totally see it on my machine (see below). you may have to reload the page, or clear your cache or something? 🤷‍♂️
Screen Shot 2022-02-03 at 10 34 11

**Comment by shanlyMIDE** [comment]
Nice, it's there now! I need to be more patient with `read the docs`! Thanks!

**Pull request merged**